pt-BR/mail/chrome/messenger/preferences/display.dtd

```diff
@@ -29,64 +29,64 @@
 <!ENTITY defaultSize.accesskey   "S">
 <!ENTITY fontOptions.accesskey   "A">
 <!ENTITY fontOptions.label       "Avançado…">
 
-<!ENTITY pb.messagelist.colortagrow.label "Apply the topic color to the entire row in the message list pane">
+<!ENTITY pb.messagelist.colortagrow.label "Aplique a cor na linha inteira no painel de lista de mensagens">
 <!ENTITY pb.messagelist.colortagrow.accesskey "g">
 
 <!-- conversation preferences -->
-<!ENTITY pb.itemConversation.label        "Conversations">
-<!ENTITY pb.conversations.display.label "Include messages from:">
-<!ENTITY pb.conversations.folderonly.label1 "The current folder">
-<!ENTITY pb.conversations.all.label1 "Every folder">
+<!ENTITY pb.itemConversation.label        "Conversas">
+<!ENTITY pb.conversations.display.label "Incluir mensagens de:">
+<!ENTITY pb.conversations.folderonly.label1 "Pasta atual">
+<!ENTITY pb.conversations.all.label1 "Todas as pastas">
 
-<!ENTITY pb.conversations.sortorder.label "Order conversations by:">
-<!ENTITY pb.conversations.sortoldest.label "Oldest messages first">
-<!ENTITY pb.conversations.sortnewest.label "Newest messages first">
+<!ENTITY pb.conversations.sortorder.label "Ordenar conversas pelas:">
+<!ENTITY pb.conversations.sortoldest.label "Mensagens mais antigas ptimeiro">
+<!ENTITY pb.conversations.sortnewest.label "Mensagens mais novas primeiro">
 
-<!ENTITY pb.conversations.read.label "When displaying a converation, mark its messages as read">
+<!ENTITY pb.conversations.read.label "Ao exibir uma conversa, marque as mensagens como lidas">
 <!ENTITY pb.conversations.read.accesskey "c">
-<!ENTITY pb.conversations.photos.label "Include contact photos for each participant in the conversation">
+<!ENTITY pb.conversations.photos.label "Incluir fotos de contatos de cada participante da conversa">
 <!ENTITY pb.conversations.photos.accesskey "I">
 
 
 <!--
 entities below were automatically added by l10nmerge
 -->
 
-<!ENTITY pbChromeFontSize.label           "Message List and Folder Font Size:">
-<!ENTITY useRetina.label             "Retina Display">
-<!ENTITY pbChromeFontSizeMedium.label     "Medium">
-<!ENTITY pbChromeFontSizeSmall.label      "Small">
-<!ENTITY pbChromeFontSizeLarge.label      "Large">
-<!ENTITY pb.itemGeneral.label        "General">
+<!ENTITY pbChromeFontSize.label           "Tamanho da fonte da lista de mensagens e da pasta:">
+<!ENTITY useRetina.label             "Tela de retina">
+<!ENTITY pbChromeFontSizeMedium.label     "Médio">
+<!ENTITY pbChromeFontSizeSmall.label      "Pequeno">
+<!ENTITY pbChromeFontSizeLarge.label      "Grande">
+<!ENTITY pb.itemGeneral.label        "Geral">
 <!ENTITY pbChromeFontSize.accesskey       "M">
 
 
 <!--
 entities below were automatically added by l10nmerge
 -->
 
-<!ENTITY quotedColor2.label "Quoted color, level 2">
-<!ENTITY devPixelsPerPx.label        "High DPI scaling:">
+<!ENTITY quotedColor2.label "Cor da citação, nível 2">
+<!ENTITY devPixelsPerPx.label        "Alta escala de DPI:">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY devPixelsPerPx.label        "Escala de DPI:">